src/translations/en.json

```diff
@@ -3166,8 +3166,8 @@
                   "label": "Numeric state",
                   "above": "Above",
                   "below": "Below",
-                  "mode_above": "Above mode",
-                  "mode_below": "Below mode",
+                  "mode_above": "Lower limit",
+                  "mode_below": "Upper limit",
```

Comment:
I think we should rename the key names if we are going to use them as a description for the entire section.

Reply:
Sure makes sense for consistency. Then we need to also rename them for the condition as well as this references the labels from the trigger. Shall I go ahead with that?

Reply:
Yes please 👍

Reply:
I made the additional changes in the strings.json but I'm afraid that the necessary changes in

ha-automation-trigger-numeric_state.ts
ha-automation-condition-numeric_state.ts

are beyond my rudimentary  programming skills and what the GitHub web editor allows me to mess with …

Reply:
All checks have passed, but the changes are quite extensive so I can't judge if that is going to break something else.